wis2node/log.py

```diff
@@ -0,0 +1,61 @@
+###############################################################################
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+###############################################################################
+
+"""Logging system"""
+
+import logging
+import sys
+
+LOGGER = logging.getLogger(__name__)
+
+
+def setup_logger(loglevel, logfile=None):
```

Comment:
Add function annotation:

`def setup_logger(loglevel: str = None, logfile: str = None) -> None:`

Reply:
Thinking more, perhaps our default should be logging to STDOUT in ERROR mode (matches the default settings in `wis2node/env.py`).  So:

`def setup_logger(loglevel: str = 'ERROR', logfile: str = stdout) -> None:`